sky.material.uniforms property do not exist on type Material

I am using angular9 and threejs with it's Sky model in itself example.

```
import { Sky } from "three/examples/jsm/objects/Sky";
...
        var sky = new Sky();
        var uniforms = sky.material.uniforms;

```
when I compplie the code with `ng server`

the compiler gave the complain:

_
: Compiled successfully.
    
    ERROR in src/app/app.component.ts:108:37 - error TS2339: Property 'uniforms' does not exist on type 'Material | Material[]'.
      Property 'uniforms' does not exist on type 'Material'.
    
    108         var uniforms = sky.material.uniforms;
                                            ~~~~~~~~
    src/app/app.component.ts:137:26 - error TS2339: Property 'uniforms' does not exist on type 'Material | Material[]'.
      Property 'uniforms' does not exist on type 'Material'.
    
    137             sky.material.uniforms[ 'sunPosition' ].value = light.position.copy( light.position );
_


##### Three.js version

"three@0.111.0"

##### Node version

v13.5.0

##### Angular version

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0-rc.7
@angular-devkit/build-angular     0.900.0-rc.7
@angular-devkit/build-optimizer   0.900.0-rc.7
@angular-devkit/build-webpack     0.900.0-rc.7
@angular-devkit/core              9.0.0-rc.7
@angular-devkit/schematics        9.0.0-rc.7
@ngtools/webpack                  9.0.0-rc.7
@schematics/angular               9.0.0-rc.7
@schematics/update                0.900.0-rc.7
rxjs                              6.5.3
typescript                        3.6.4
webpack                           4.41.2

--- a/examples/jsm/objects/Sky.d.ts
+++ b/examples/jsm/objects/Sky.d.ts
@@ -1,10 +1,15 @@
 import {
-	Mesh
+	BoxBufferGeometry,
+	Mesh,
+	ShaderMaterial
 } from '../../../src/Three';
 
 export class Sky extends Mesh {
 
 	constructor();
+
+	geometry: BoxBufferGeometry;
+	material: ShaderMaterial;
 
 	static SkyShader: object;
 

